tests/virtual_machines_test.go

```diff
@@ -217,8 +238,7 @@ var _ = Describe("Virtual Machines", func() {
 				Expect(err).To(HaveOccurred())
 			})
 		})
-		//2165 test postponed due to issue: https://github.com/k8snetworkplumbingwg/kubemacpool/issues/105
-		PContext("when trying to create a VM after a MAC address has just been released duo to a VM deletion", func() {
+		Context("when trying to create a VM after a MAC address has just been released duo to a VM deletion", func() {
```

Comment:
Please keep `//2165` so we have a reference what to replace later with proper polarion ids.